src/frontend/src/components/ProposedSolutionView.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { ProposedSolution } from 'shared';
+
+import PageBlock from '../layouts/PageBlock';
+import { Badge, Col, Container, Row } from 'react-bootstrap';
+import { dollarsPipe, weeksPipe } from '../utils/Pipes';
+import { useTheme } from '../hooks/Theme.hooks';
+
+interface ProposedSolutionViewProps {
+  proposedSolution: ProposedSolution;
+  selected: boolean;
+  setter: any;
+}
+
+const ProposedSolutionView: React.FC<ProposedSolutionViewProps> = ({
+  proposedSolution,
+  selected,
+  setter
+}) => {
+  const styles = {
+    white: {
+      color: 'white'
+    },
+    black: {
+      color: 'black'
+    }
+  };
+
+  const theme = useTheme().name === 'DARK' ? styles.white : styles.black;
+
+  const spacer = 'mb-1';
```

Comment:
the theme part or the spacer part? the only reason i don't do this with the theme stuff is because this isn't extensible to more than two themes. If it's the spacer stuff then yeah, and you coudl make a lot of easy tickets out of it which would be useful

Reply:
the spacer part, I'll make a note somewhere to make a bunch of those tickets @anthonybernardi